.gitmodules

```diff
@@ -14,5 +14,5 @@
 	path = android/app/src/main/cpp/oboe
 	url = https://github.com/google/oboe
 [submodule "third-party/borealis"]
```

Comment:
the submodule still refer the "third-party" path in its title. 
I do not know if it's intended or not. 
(git submodules can be tricky to move or delete sometimes)